source/GameData.cpp

```diff
@@ -1260,8 +1260,9 @@ void GameData::PrintShipTable()
 void GameData::PrintWeaponTable()
 {
 	cout << "name" << '\t' << "cost" << '\t' << "space" << '\t' << "range" << '\t'
-		<< "energy/s" << '\t' << "heat/s" << '\t' << "shield/s" << '\t' << "hull/s" << '\t'
-		<< "homing" << '\t' << "strength" << '\n';
+		<< "energy/s" << '\t' << "heat/s" << '\t' << "move/s" << '\t'
```

Comment:
"move" seems non-intuitive. I wonder if we need to provide description lines before or after the table, to enable us to use terse / unintuitive column headers.

Reply:
Maybe "recoil"?

I prefer not to add lines before or after the table itself; the current table exports nicely into a tsv (tab separated values) file by using `endless-sky --weapons > weapons.tsv` and can then easily be imported into Excel or OpenOffice Calc.
If we want to give descriptions, then a markdown file for the wiki might be more appropriate (or we could switch from terse headers to less terse headers).

Reply:
IMO, header & footer rows don't harm the ability to import data into spreadsheet software, especially when the data delimiter isn't common punctuation.

If we introduce them, I recommend we use header rows over footer rows, so the column descriptors can be visible when the column headers are. Probably we do not need to annotate all column names, either.

Reply:
For now I changed the text to `recoil` to keep the header tense but hopefully less unintuitive.